logbook-core/src/main/java/org/zalando/logbook/core/package-info.java

```diff
@@ -1,4 +1,2 @@
-@ParametersAreNonnullByDefault
 package org.zalando.logbook.core;
 
-import javax.annotation.ParametersAreNonnullByDefault;
```

Comment:
I vaguely remember this added some compile-time null-safety guarantees.by adding nonnull annotation to all parameters, unless specified otherwise. (i.e., via `@Nullable`).

I couldn't readily find a replacement; but would be great to check more.